tests/ui_tests/test_service_page.py

```diff
@@ -0,0 +1,403 @@
+    """Upload provider bundle"""
+    return sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), request.param))
+
+
+@pytest.fixture()
+@allure.title("Create provider")
+def upload_and_create_provider(provider_bundle) -> Provider:
+    """Create provider from uploaded bundle"""
+    return provider_bundle.provider_create(PROVIDER_NAME)
+
+
+@pytest.fixture()
+@allure.title("Create community cluster and add host")
+def create_community_cluster_with_host(app_fs, sdk_client_fs: ADCMClient, upload_and_create_provider, create_host):
+    """Create community cluster and add host"""
+    bundle = cluster_bundle(sdk_client_fs, BUNDLE_COMMUNITY)
+    cluster = bundle.cluster_create(name=CLUSTER_NAME)
+    host = cluster.host_add(create_host)
+    return cluster, host
+
+
+@pytest.fixture()
+@allure.title("Create community cluster with service and add host")
+def create_community_cluster_with_host_and_service(sdk_client_fs: ADCMClient, create_host):
+    """Create community cluster with service and add host"""
+    bundle = cluster_bundle(sdk_client_fs, BUNDLE_COMMUNITY)
+    cluster = bundle.cluster_create(name=CLUSTER_NAME)
+    return cluster, cluster.service_add(name=SERVICE_NAME), cluster.host_add(create_host)
+
+
+@pytest.fixture()
+@allure.title("Create host")
+def create_host(upload_and_create_provider):
+    """Create default host using API"""
+    provider = upload_and_create_provider
+    return provider.host_create(HOST_NAME)
+
+
+@pytest.fixture()
+def create_import_cluster_with_service(sdk_client_fs: ADCMClient):
+    """Create clusters and services for further import"""
+    params = {
+        "import_cluster_name": "Import cluster",
+        "import_service_name": "Pre-uploaded Dummy service to import",
+    }
+    with allure.step("Create main cluster"):
+        bundle = cluster_bundle(sdk_client_fs, BUNDLE_COMMUNITY)
+        cluster_main = bundle.cluster_create(name=CLUSTER_NAME)
+        service_main = cluster_main.service_add(name=SERVICE_NAME)
+    with allure.step("Create cluster to import"):
+        bundle = cluster_bundle(sdk_client_fs, BUNDLE_IMPORT)
+        cluster_import = bundle.cluster_create(name=params["import_cluster_name"])
+        service_import = cluster_import.service_add(name=params["import_service_name"])
+        return cluster_main, service_main, cluster_import, service_import
+
+
+class TestServiceMainPage:
+    """Tests for the /cluster/{}/service/{}/main page"""
+
+    @pytest.mark.smoke()
+    def test_open_service_main_page_by_tab(self, app_fs, create_community_cluster_with_service):
+        """Test open /cluster/{}/service/{}/main page from left menu"""
+
+        cluster, service = create_community_cluster_with_service
+        service_config_page = ServiceConfigPage(app_fs.driver, app_fs.adcm.url, cluster.id, service.id).open()
+        service_main_page = service_config_page.open_main_tab()
+        service_main_page.wait_page_is_opened()
```

Comment:
Don't we want to wait for page inside `open_{}_tab`?

Reply:
fixed